src/app/routes/utils/constructPageFetchUrl/index.test.ts

```diff
@@ -96,6 +96,9 @@ describe('constructPageFetchUrl', () => {
     ${LIVE_PAGE}    | ${null}         | ${null}    | ${'local'}  | ${'c0000000000t'}                                     | ${'http://localhost/api/local/ukrainian/live/c0000000000t'}
     ${LIVE_PAGE}    | ${null}         | ${null}    | ${'test'}   | ${'c0000000000t'}                                     | ${'https://mock-bff-path/?id=c0000000000t&service=ukrainian&pageType=live&serviceEnv=test'}
     ${LIVE_PAGE}    | ${null}         | ${null}    | ${'live'}   | ${'c0000000000t'}                                     | ${'https://mock-bff-path/?id=c0000000000t&service=ukrainian&pageType=live&serviceEnv=live'}
+    ${LIVE_PAGE}    | ${'arabic'}     | ${null}    | ${'local'}  | ${'67574192'}                                         | ${'http://localhost/api/local/arabic/live/67574192'}
```

Comment:
Could add a check for variant languages here for TIPO/CPS assets?

Reply:
@amoore108 We have found a CPS live page test url example which I have added in a test case for, but struggling to find other variant example assets. Do you have any to hand, or able to direct me to somewhere I will be able to find some more? 😄

Reply:
There won't be any real Test/Live TIPO assets unfortunately, but you should be able to just use something like `c0000000000t` in the other examples to mock what a TIPO asset would look like.

Reply:
Ah of course, thanks 🙂